File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/stock/StockForm.java
package com.isystk.sample.web.front.controller.api.v1.stock;

import com.isystk.sample.common.dto.Pageable;
import com.isystk.sample.web.base.controller.html.BaseSearchForm;

public class StockForm extends BaseSearchForm implements Pageable {

  private static final long serialVersionUID = 7593564324192730932L;

  int perpage = 6;

  public int getPerpage() {
    return this.perpage;
  }

  public void setPerpage(int perpage) {
    this.perpage = perpage;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/stock/StockForm.kt
package com.isystk.sample.web.front.controller.api.v1.stock

import com.isystk.sample.common.dto.Pageable
import com.isystk.sample.web.base.controller.html.BaseSearchForm

class StockForm : BaseSearchForm(), Pageable {
    override fun perpage(): Int {
        return 6
    }

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}